### PR TITLE
[TwigBridge] Add types to private properties

### PR DESCRIPTION
**Pull request by derrabus**
| Q             | A
| ------------- | ---
| Branch?       | 6.0
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | N/A
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
(a test is failing: "TwigDataCollector::$profile must not be accessed before initialization")

**Comment by derrabus** [comment]
Should be fixed now.

**Comment by derrabus** [comment]
The Psalm failures are false positives.

File: src/Symfony/Bridge/Twig/Mime/BodyRenderer.php [hunk 1]
```diff
@@ -22,9 +22,9 @@
  */
 final class BodyRenderer implements BodyRendererInterface
 {
-    private $twig;
-    private $context;
-    private $converter;
+    private Environment $twig;
+    private array $context;
+    private HtmlConverter $converter;
```
Comment (stof) [inline src/Symfony/Bridge/Twig/Mime/BodyRenderer.php:27]: I'd mark that nullable, as it is an optional dependency and so not having a converter is a normal state

**Comment by nicolas-grekas** [comment]
Thank you @derrabus.

File: src/Symfony/Bridge/Twig/Mime/BodyRenderer.php (hunk 1 above)
Comment (derrabus) [reply src/Symfony/Bridge/Twig/Mime/BodyRenderer.php:27]: Leaving it uninitialized also works well in this case, but I see your point. Now that the PR is merged, would you like to change the property to nullable? I'd accept a PR with that change. 🙂

**Pull request merged**